unity-client/Assets/Scripts/MainScripts/DCL/Controllers/StickersController/StickersController.cs

```diff
@@ -0,0 +1,20 @@
+﻿using UnityEngine;
+
+public class StickersController : MonoBehaviour
+{
+    private StickersFactory stickersFactory;
+
+    private void Awake() { stickersFactory = Resources.Load<StickersFactory>("StickersFactory"); }
+
+    public void PlayEmote(string id)
+    {
+        if (stickersFactory == null || !stickersFactory.TryGet(id, out GameObject prefab))
+            return;
+
+        GameObject emoteGameObject = Instantiate(prefab);
```

Comment:
Should we make this instance in a pool? I am thinking about lots of users watching a concert and showing emotes to their artist. What do you think?

Reply:
Yeah, I want to leverage the performance before implementing the pool and I need some stickers to be implemented before hand. What do you think about coming back to this topic once they are set?

Reply:
Perfect!